core/controllers/base_test.py

```diff
@@ -463,6 +463,13 @@ def _mock_logging_function(msg, *args):
 
         self.assertEqual(observed_log_messages, ['Frontend error: errors'])
 
+    def test_redirect_oppia_test_server(self):
+        # The old demo server redirects to the new demo server.
+        response = self.get_html_response(
+            'https://oppiaserver.appspot.com/splash', expected_status_int=301)
```

Comment:
Is this something we want to hardcode? I'm thinking of the case where there are other instances of our oppia server or if that old test server is down. 

/cc @seanlip

Reply:
I think it's OK in this case. There was a similar test existing before, and we do special-case this in the main code file that is being tested here.

Reply:
Ah, okay.